lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -11,9 +11,12 @@ defmodule ExUnit.Case do
 
   When used, it accepts the following options:
 
-    * :async - configure this specific test case to able to run in parallel
+    * `:async` - configure this specific test case to able to run in parallel
```

Comment:
Doesn't it depend? It is ok to use async as long as you understand that this will happen and you are ok with it.